scripts/ci/_common.sh

```
@@ -0,0 +1,366 @@
+
+sha256_stream() {
+  if command -v sha256sum >/dev/null 2>&1; then
+    sha256sum
+  else
+    shasum -a 256
+  fi
+}
+
+repo_relative_path() {
+  local path="$1"
+  printf '%s\n' "${path#${REPO_ROOT}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)